guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,325 @@
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Combining-channel-opens-mobile@2x.png"
+   alt-text = "Graphic depicting combining a collaborative fund and zero-confirmation channel open technique"
+   width = 800
+   height = 348
+%}
+
+## Where do LSPs offer this service?
+
+As mentioned above, LSPs makes it easy for users to connect to and use lightning by offering inbound liquidity as a service. There are various ways LSPs offer this service which range in complexity. From being in-built into wallets like [Breez](https://breez.technology/), web portals like [Blocktank](https://synonym.to/blocktank/), P2P marketplaces like [Magma](https://amboss.space/magma), to protocol level methods such as [liquidity advertisements](https://bitcoinops.org/en/topics/liquidity-advertisements/).
+
+{% include tip/open.html color="blue" icon="info" label="Avoid LSP lock-in" %}
+
+If your application uses an LSP ensure users can switch, opt out, or use multiple LSPs. This ensures they have choice and can choose the LSPs that have the trade offs they are comfortable with.
+
+It also prevents everyone using the same LSP, improving [decentralization]({{ '/guide/getting-started/principles/#decentralization' | relative_url }}).
+
+{% include tip/close.html %}
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/where-is-this-serviced-offered.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/where-is-this-serviced-offered@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/where-is-this-serviced-offered-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/where-is-this-serviced-offered-mobile@2x.png"
+   alt-text = "Graphic depicting where LSP services are offered"
+   width = 800
+   height = 348
+%}
+
+## What are Lightning wallet servers?
+
+Once a user is connected to the lightning network, likely through an LSP, lightning wallet servers (LWS) offer services that make using lightning easier for users. LSPs sometimes offer LWS services as part of their offering but this is not exclusive to LSPs.
+
+LWSs act as trust-minimized third-parties, so when possible users should be able to opt-out and operate their lightning wallet manually.
+
+Below are some LWS services and the user friction points they solve.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/what-are-lightning-wallet-servers.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/what-are-lightning-wallet-servers@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/what-are-lightning-wallet-servers-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/what-are-lightning-wallet-servers-mobile@2x.png"
+   alt-text = "Graphic depicting what an LWS offers users"
+   width = 800
+   height = 348
+%}
```

Comment:
Recommended text:

- User - Finds it hard to connect to lightning
- LSP - Makes it easy to connect to lightning
- LWS - Once on lightning, helps solve other usability issues

The current phrasing makes it sound like the LWS enables _the users to solve their own usability issues_. I don't think that's the case. What we really need to convey is that the LWS enables product designers and developers to solve these usability issues on behalf of users. So just simply to "helps solve other usability issues".